### DISCUSSION TITLE
Allow using only a `fileSlug` in `inputpath-to-url`, in addition to full path

### DISCUSSION THREAD

**Discussion by nhoizey** [11ty/eleventy#4138] [Enhancement Queue]
### Describe the solution you'd like

https://www.11ty.dev/docs/plugins/inputpath-to-url/ is a great addition to Eleventy features.

Would it be possible to enhance it to be able to use only the `fileSlug` in the link, instead of the full path, if we are sure there’s only one such fileSlug?

That's what this plugin provides: https://github.com/nhoizey/eleventy-plugin-link_to

**Comment by zachleat** [comment]
```html
<a href="my-template.md">Home</a> and <a href="my-template">Home</a>
```
